framework/include/postprocessors/FindValueOnLine.h

```diff
@@ -48,6 +48,12 @@ class FindValueOnLine : public GeneralPostprocessor, public Coupleable
   /// value to find along the line
   const Real _target;
 
+  /// boolean indicating whether to stop with an error if value is not found on the line
+  const bool _stop_if_not_found;
+
+  /// value to return if target value is not found on the line and _stop_if_not_found is false
+  const Real _not_found_val;
```

Comment:
```suggestion
  const & bool _stop_if_not_found;

  /// value to return if target value is not found on the line and _stop_if_not_found is false
  const & Real _not_found_val;
```

Reply:
This will allow these parameters to be declared as "controllable"in the future, if needed.